lib/pdf.js

```diff
@@ -85,7 +85,7 @@ PDF.prototype.toFile = function PdfToFile (filename, callback) {
 
 PDF.prototype.exec = function PdfExec (callback) {
   var callbacked = false
-  var child = childprocess.spawn(this.options.phantomPath, [].concat(this.options.phantomArgs, [this.script]))
+  var child = childprocess.spawn(this.options.phantomPath, [].concat(this.options.phantomArgs, [this.script]), this.options.cliOptions)
```

Comment:
No. `phantomArgs` refers to the command-line arguments, which should be passed to PhantomJS. This `cliOptions` refers spawn optons, which way the childprocess should be called. See: [Node childprocess options](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options)